geomesa-feature/pom.xml

```diff
@@ -36,9 +36,9 @@
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.esotericsoftware</groupId>
-            <artifactId>kryo-shaded</artifactId>
-            <version>3.0.0</version>
+            <groupId>com.esotericsoftware.kryo</groupId>
+            <artifactId>kryo</artifactId>
+            <version>2.21</version>
```

Comment:
did you make sure that kryo 3 isn't coming in from somewhere else? i thought that there was a line of code that would have to be changed for kryo 2...